### PR TITLE
WM4_Kavita Patil_JavaScript_Core-2_Week-4

### PR DESCRIPTION
**Pull request by kavitappatil**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: 1-mandatory/1-alarmclock/alarmclock.js [hunk 1]
```diff
@@ -1,4 +1,30 @@
-function setAlarm() {}
+function setAlarm() {
+  let alarmSetValue = document.getElementById("alarmSet").value;
```
Comment (Pat-On) [inline 1-mandatory/1-alarmclock/alarmclock.js:2]: 👍

File: 1-mandatory/2-quotegenerator/quotes.js [hunk 2]
```diff
@@ -1,3 +1,26 @@
+let quoteEl=document.getElementById("quote_text");
+let nameEl=document.getElementById("writer_name");
+let button=document.getElementById("new_quote");
+let bodyEl=document.getElementById("content")
+button.addEventListener("click", () => {
```
Comment (Pat-On) [inline 1-mandatory/2-quotegenerator/quotes.js:5]: Nicely done. Do You have auto-formatting on?

File: 1-mandatory/3-slideshow/slideshow.js [hunk 3]
```diff
@@ -1 +1,53 @@
 // Write your code here
+let imageArray = ['image/photo-1.avif', 'image/photo-2.avif','image/photo-3.avif','image/photo-4.avif','image/photo-5.avif']
```
Comment (Pat-On) [inline 1-mandatory/3-slideshow/slideshow.js:2]: Great job :)) 
image

**Review by Pat-On — COMMENTED** [review]
Great job! You are doing very well. Please keep studying hard :) 🥇

File: 1-mandatory/2-quotegenerator/quotes.js (hunk 2 above)
Comment (kavitappatil) [reply 1-mandatory/2-quotegenerator/quotes.js:5]: yes, before submitting my work, I am formatting my files. thank you, Patryk

**Pull request closed without merging**